src/instance/misc.js

```diff
@@ -39,6 +39,14 @@ exports._applyFilters = function (value, oldValue, filters, write) {
   return value
 }
 
+exports._bubbleChanges = function (value, expression) {
```

Comment:
I think splitting this here makes the code harder to follow. It's ok that this check stay in the watcher.